test/test.js

```diff
@@ -64,8 +71,19 @@ describe('cat [ by lines ]', function() {
 });
 
 describe('bc -l [ by lines ]', function() {
-  const instance = new curlable.Curlable('bc -l');
-  instance.registerRoutes(app, '/bc');
+  let instance;
+  let app;
+
+  before(function() {
```

Comment:
Out of curiosity: not using `() => { ... }` here to keep `this` intact?

Reply:
Yes, it's the `mocha` API which passes context through `this`, not through an arg. Your friend's `suman` doesn't do that, but I haven't tried it yet.

In this particular case it doesn't matter because `this` isn't referenced.